WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -77,6 +77,14 @@ $(function() {
     }
   });
 
+  // We use dp.hide to capture clicking on a date. We then tab to the next
+  // form item, so the picker is just one click away, in case the user made
+  // a mistake or wants to select a different date. This also triggers the
+  // 'blur' event, used above to validate the date.
+  $datetimepicker.on('dp.hide', function() {
+    $('input:not(input[type=button], input[type=submit], input[type=reset]), select, textarea').eq(parseInt($('input:not(input[type=button], input[type=submit], input[type=reset]), select, textarea').index($(this)), 10) + 1).focus();
```

Comment:
Like this:
```Javascript
var focusables = $(":focusable");   
var current = focusables.index(this),
next = focusables.eq(current+1).length ? focusables.eq(current+1) : focusables.eq(0);
next.focus();
```

There is a problem. With datetimepickers, it will not hide once you click the date. Or when you choose a time. It will never hide until you click outside :P Not sure what to do with that.